src/auto_scheduler/compute_dag.cc

```diff
@@ -342,11 +343,16 @@ AccessAnalyzer::AccessAnalyzer(const Array<te::Tensor>& tensors) {
         has_expensive_op |= HasExpensiveOp(expr);
       }
       if (has_expensive_op || has_branch[op]) {
-        is_strict_inlineable = false;
+        is_strictly_inlineable = false;
+      }
+
+      // constant tensor is strict-inlineable
+      if (node->read_from[op].empty()) {
+        is_strictly_inlineable = true;
       }
```

Comment:
@merrymercy The transform_matrices A, B of the winograd conv2d is a constant tensor but with branch, so the order of `has_branch[op]` and `read_from[op]` check will result on different `is_strictly_inlineable`.
Should this intend to be strictly inlined?

Reply:
We should always inline const matrix of transform matrices in Winograd. However, like `data_pad` which has `branch` but not `constant`, we shouldn't always inline it and should let it try different computation location to get better performance by solving `parallel-locality-compute` trade off. So I think this logic here makes sense to me.

Reply:
@FrozenGene is correct. The later rules have higher priority and overwrite previous ones.